tests/test-docregistry/src/context.spec.ts

```diff
@@ -142,14 +141,15 @@ describe('docregistry/context', () => {
         await context.initialize(false);
         await context.ready;
         expect(context.model.cells.canUndo).to.be(false);
+        await dismissDialog();
```

Comment:
I think that this may not be necessary now that there is an `await` above (from #4895). The dialog is spurious due to a race condition, no?

Reply:
Good call. It's gone now.